### ISSUE TITLE
Addon disappeared from mozilla addons

### ISSUE THREAD

**Issue by viktorstrate** [Benjadahl/UDPlus-Plus#81]
The extension was avaliable at the link below. But the addon can't be found anymore.

https://addons.mozilla.org/en-US/firefox/addon/udplus-plus/versions/

**Comment by jbjjbjjbj** [comment]
Yeah i know this, they didn't like the fact that we had the full library folders and not just the .min files. 

I tried fixing it but it kept giving me an error so i tried removing and re-releasing the extension, but that can't be done before we make a new release.

I will try and clean up the lib folder for duplicates.

**Comment by viktorstrate** [comment]
I will be happy to setup a build system, using nodejs and webpack.
This would make it possible to automattically generate a minified and optimized javascript file.
This could enable _hot reloading_, and support for ES7 features.

**Comment by jbjjbjjbj** [comment]
Hot reloading?
Well if you want to that would be cool, would that also work for the chrome store?

**Comment by viktorstrate** [comment]
It would also work for chrome :)
Hot reloading means that as soon as you save a file, the plugin wil get updated live in the browser.

**Comment by jbjjbjjbj** [comment]
Ohh thats cool. It's fine with me.

I'm releasing v2.3.1 using the old method, to get it back on the firefox store as fast as possible

**Comment by viktorstrate** [comment]
Nice, thank you for the release update 👍

**Comment by jbjjbjjbj** [comment]
It's up again

https://addons.mozilla.org/en-US/firefox/addon/udplusplus/
But a better release method is welcome

**Comment by jbjjbjjbj** [comment]
Sooo the plugin was just pulled again from the store. This time with the comment that our audience is to limited :-(

We will have to self host the extension instead :-1:

Response from firefox:


> Details:
> 
> We are sorry but due to limited audience this add-on is not suitable for being listed. Please submit a self-hosted version and distribute the signed file yourself. For more information, see https://developer.mozilla.org/en-US/Add-ons/Distribution and https://developer.mozilla.org/en-US/Add-ons/AMO/Policy/Reviews#Policies. Please read the content section.

**Comment by viktorstrate** [comment]
We could upload the signed extension to github, in a [release](https://github.com/Benjadahl/UDPlus-Plus/releases)?

**Comment by jbjjbjjbj** [comment]
Jep thats the plan. The problem is that the users will have no idea when a new version is out. We could make a checker but then we would have to alter the domain permissions.
